### PR TITLE
fix numeric lists in ci/cd docs

### PR DESCRIPTION
**Pull request by matanper**

### CODE REVIEW

File: docs/source/user-guide/general/ci_cd.rst [hunk 1]
```diff
@@ -2,60 +2,62 @@
 Using Deepchecks In CI/CD
 =================================
 
-This guide will explain the basics of using CI/CD for machine learning, and how deepchecks can be incorporated into
-the process.
+This guide will explain the basics of using CI/CD for machine learning, and demonstrate how deepchecks
```
Comment (noamzbr) [inline docs/source/user-guide/general/ci_cd.rst:5]: Sentence became a bit too long

File: docs/source/user-guide/general/ci_cd.rst [hunk 2]
```diff
@@ -2,60 +2,62 @@
...
-  :doc:`duplicate samples between the datasets</checks_gallery/tabular/train_test_validation/plot_train_test_samples_mix>`,
-  and more.
-* Model training: The model is trained on the (previously validated) training set.
-* Model validation: The trained model is evaluated on test data, testing for performance, weak segments and more:
+software products. These can be made more trustworthy and efficient, and help improve the R&D processes,
+by adding automated steps to the development and deployment lifecycle.
+For ML models, CI/CD concepts can be utilized to streamline the process of model training
+(and retraining), data and model validation and model deployment:
+
+* | **Data integrity validation**: When the data used for training is collected via automatic processes and pipelines,
```
Comment (noamzbr) [inline docs/source/user-guide/general/ci_cd.rst:24]: What does this do?

**Review by noamzbr — APPROVED** [review]

File: docs/source/user-guide/general/ci_cd.rst (hunk 2 above)
Comment (matanper) [reply docs/source/user-guide/general/ci_cd.rst:24]: it enables a new line which is still under the same bullet

File: docs/source/user-guide/general/ci_cd.rst (hunk 1 above)
Comment (matanper) [reply docs/source/user-guide/general/ci_cd.rst:5]: suggestion?

**Pull request merged**